docs/docs/20-quickstart.mdx

```diff
@@ -300,7 +300,7 @@ the previous section.
 
 1. Next, we'll create several Kargo resource:
 
-    1. A `Project`, which, when reconciled, will effect all boilerplate project
+    1. A `Project`, which, when reconciled, will affect all boilerplate project
```

Comment:
"effect" was actually correct here.

https://www.merriam-webster.com/grammar/affect-vs-effect-usage-difference

Sorry... there's no convenient anchor on that page, but refer to the "A Few Rare Exceptions" section.